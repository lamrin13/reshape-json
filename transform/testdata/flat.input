{
    "name": "John",
    "email": "john.doe@abc.com",
    "phone": "6471663421"
}