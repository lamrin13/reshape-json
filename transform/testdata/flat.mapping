{
    "firstname":    "name",
    "emailAddress": "email",
    "phoneNumber":  "phone"
}