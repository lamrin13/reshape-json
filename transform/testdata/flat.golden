{
    "firstname":"John",
    "emailAddress":"john.doe@abc.com",
    "phoneNumber":"6471663421"
}
